--- a/webui/vue.config.js
+++ b/webui/vue.config.js
@@ -1,5 +1,5 @@
 module.exports = {
-  publicPath: '/',
+  publicPath: '/vertex/',
   outputDir: '../app/static',
   assetsDir: 'assets',
   devServer: {
@@ -48,7 +48,7 @@ module.exports = {
     manifestOptions: {
       name: 'VERTEX',
       short_name: 'VERTEX',
-      start_url: '/',
+      start_url: '/vertex/',
       display: 'standalone',
       icons: require('./public/assets/pwaicons/icons.json').icons
     },
